File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/SetOuttakeTiltServoCommand.java
package org.firstinspires.ftc.teamcode.commands.low_level;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

public class SetOuttakeTiltServoCommand extends InstantCommand {
    public SetOuttakeTiltServoCommand(OuttakeManager manager, OuttakeManager._OuttakeTiltServoState state) {
        super(() -> manager.update(state));
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/low_level/outtake/SetOuttakeTiltServoCommand.kt
package org.firstinspires.ftc.teamcode.commands.low_level.outtake;

import com.arcrobotics.ftclib.command.InstantCommand;

import org.firstinspires.ftc.teamcode.managers.OuttakeManager;

class SetOuttakeTiltServoCommand(manager: OuttakeManager, state: OuttakeManager._OuttakeTiltServoState) : InstantCommand({manager.update(state)})